packages/angular/src/generators/ngrx/lib/add-imports-to-module.ts

```diff
@@ -7,16 +7,16 @@ import {
   addImportToModule,
   addProviderToModule,
 } from '../../../utils/nx-devkit/ast-utils';
-import type { NgRxGeneratorOptions } from '../schema';
+import type { NormalizedNgRxGeneratorOptions } from './normalize-options';
 
 export function addImportsToModule(
   tree: Tree,
-  options: NgRxGeneratorOptions
+  options: NormalizedNgRxGeneratorOptions
 ): void {
-  const modulePath = options.module;
-  const sourceText = tree.read(modulePath, 'utf-8');
+  const parentPath = options.module ?? options.parent;
```

Comment:
nit: since both module and parent are optional now but I think we have to have at least either one. `parent` will be made required as we remove `module` in the future but can/should we put a check for them here, and log an actionable error message for now?